src/kagglehub/models.py

```diff
@@ -13,22 +14,44 @@
 DEFAULT_IGNORE_PATTERNS = [".git/", "*/.git/", ".cache/", ".huggingface/"]
 
 
-def model_download(handle: str, path: Optional[str] = None, *, force_download: Optional[bool] = False) -> str:
+def model_download(
+    handle: str,
+    path: Optional[str] = None,
+    *,
+    force_download: Optional[bool] = False,
+    output_dir: Optional[str] = None) -> str:
     """Download model files.
 
     Args:
         handle: (string) the model handle.
         path: (string) Optional path to a file within the model bundle.
         force_download: (bool) Optional flag to force download a model, even if it's cached.
-
+        output_dir: (str) Optional path to copy model files to after successful download.
```

Comment:
we should integrate the [default path](https://github.com/Kaggle/kagglehub/blob/b1404cc92e6b0d64f5697ba8f1403f00edfc1051/src/kagglehub/config.py#L58) here instead.

Reply:
@rosbo  - The parameters for this fn is starting to grow. Any thoughts on managing this? Should we use wrap this in a settings/configuration object instead?

Reply:
just my nitpick opinion: I don't think the explicit default path here is necessary since it circumvents the general cache behavior, which is encapsulated in the http resolver.

Reply:
We want this method to be easy to use and I feel like adding a settings object would complicate it more than anything. Users will use keyword argument for these and likely set no parameters or at most 1-2.
If we compare to `huggingface_hub.snapshot_download(...)`, we don't have many arguments... https://github.com/huggingface/huggingface_hub/blob/4011b5a2836d7bb036d8da54ed656f88bc0d2f7f/src/huggingface_hub/_snapshot_download.py#L21-L45

Reply:
sg.